controller/lib/core/sensors.cpp

```diff
@@ -71,7 +78,37 @@ static constexpr Duration VOLUME_INTEGRAL_INTERVAL = milliseconds(5);
   __builtin_unreachable();
 }
 
-Sensors::Sensors() {}
+// Tuning params for PID that calculates error in our flow measurements.
+// Calculated using the
+// https://en.wikipedia.org/wiki/Ziegler%E2%80%93Nichols_method The constants in
+// the constructor are for the "classic PID" control type, which seemed to work
+// well.
+//
+// Note that we have to choose a nominal sample period for the PID.  In reality
+// we sample the PID once per breath; the exact value we use for the sample
+// period is not important so long as it's shorter than our breath period.  If
+// we were to choose a value longer than our breath period, then the PID
+// wouldn't update on some breaths, which is not desirable!
+//
+// FLOW_PID_OUTPUT_MAX is a sanity check, preventing us from using absurdly
+// high flow corrections.  100ml/s may strike you as absurdly high, but we have
+// seen justified need to go as high as 40ml/s.
```

Comment:
Please elaborate here a bit more, in what situations have we seen that happen - cause otherwise this gives the impression that the sensors are super broken and artificial mitigation would only mask that.

Reply:
Expanded the comment to incorporate what I said above.  It is now kind of an essay, but okay.  It's important and non-obvious.